src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
Rather than destroy our existing impl, we should factor it into three grades - fluid.slider containing common definitions, fluid.slider.native and fluid.slider.jQuery containing the addons necessary to construct the two different varieties of slider.